.github/actions/setup/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
 
     - name: Install package
       shell: bash
-      run: python -m pip install .
+      run: python -m pip install .[development]
```

Comment:
I think here we want also production, don't?

Reply:
👍